integrations/slack/integration.definition.ts

```diff
@@ -25,7 +89,10 @@ export default new IntegrationDefinition({
     },
     ...sentryHelpers.COMMON_SECRET_NAMES,
   },
-  user,
+  user: {
+    tags: userTags,
```

Comment:
request: Can we bring the tags here too? It's useful to see them here in the integration defintiion one swoop, I think.

Reply:
I thought about doing it but it's actually used as well in another file